sdcm/cluster.py

```diff
@@ -1903,13 +1917,24 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo):
         """.format(mgmt_user=mgmt_user, rsa_id_dst=rsa_id_dst, rsa_id_dst_pub=rsa_id_dst_pub))  # generate ssh public key from private key.
         self.remoter.run('sudo bash -cxe "%s"' % ssh_config_script)
 
+        self.config_scylla_manager_yaml(segments_per_repair=100)
```

Comment:
isn't this something we want to control ? i.e. in  SCT configuration
also if we hardcode this value, please comment on why and what it's intended for.